app/models/graph-controller.ts

```diff
@@ -81,10 +81,10 @@ export class GraphController {
         }
 
         // TODO: ...like
+        const likeVal = new ElementValue(this.activeEdgeType, this.core.environment);
 
-        return validateEdge(this.plugin, source, destination, undefined);
+        return validateEdge(this.plugin, source, destination, likeVal);
```

Comment:
I'm pretty sure you want to use the type of the provided `like` parameter instead of the active edge type. The purpose of `like` is for the case of moving an edge, which is not necessarily the active edge type.